lecture_5/homework/app/models/reporots/single_report.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Reports
+  class SiegeReport
+    def self.check_siege_ability(building:)
```

Comment:
Models to trochę złe miejsce na taki "util", do tego typu działań genialnie nadaje się ServiceObject :)